### ISSUE TITLE
Loss cannot convergent

### ISSUE THREAD

**Issue by huiofficial** [arashno/tensorflow_multigpu_imagenet#14]
I train densenet 201 on imagenet 2012, the total epochs are about 200, but the total loss is still about 1.3. 
The top-1 and top-5 accuracy is about 0.72 and 0.91, still behind the paper's result.
Is the total loss meant to be drop to 0 eventually?
![image](https://user-images.githubusercontent.com/3870219/44301104-1f1e8e80-a344-11e8-900f-646a122a100d.png)
Another problem is that every time I retrain from the previous model, at the beginning the loss is very high and accuracy is 0, and they will be normal very quickly. As far as I am concerned, the result should be the same as the model was trained, right? Did I do anything wrong?

**Comment by arashno** [comment]
This is not the official implementation of DenseNet.
The original paper is using different learning rate and weight decay policies, different data augmentation and some different hyperparameters. Therefore, a few percent difference between the highly tuned paper results and the results you get is normal.

No, the total loss will never become 0.

About the second problem, What "transfer_mode" are you using? 0 or 1?

**Comment by huiofficial** [comment]
Thank you very much for your explanation. I use transfer_mode 1.

**Comment by arashno** [comment]
You're welcome.
transfer_mode 1 replaces the last layer of the network with a randomly initialized layer, freeze the other layers and then trains the replaced layer. transfer_mode 1 is designed for transferring to a new problem with different number of classes. Therefore, having a high loss at the beginning is normal. If you want to retrain from a previous model trained for the same problem, you should use transfer_mode 0.

**Comment by huiofficial** [comment]
I got it. Thanks a lot!!!
